spec/system/whats_new_page_spec.rb

```diff
@@ -20,18 +20,20 @@
     end
 
     describe 'with subsequent logins' do
-      before do
-        click_on 'sign-out-desktop'
+      skip 'wip' do
```

Comment:
Why is skipping necessary?

Reply:
Removed [948fa98](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/968/commits/948fa98606f6487d309ced1f57de4a46e142a44f)